pandas/_libs/groupby_helper.pxi.in

```diff
@@ -293,10 +293,10 @@ def get_dispatch(dtypes):
 
 @cython.wraparound(False)
 @cython.boundscheck(False)
-def group_last_{{name}}(ndarray[{{c_type}}, ndim=2] out,
-                        ndarray[int64_t] counts,
-                        ndarray[{{c_type}}, ndim=2] values,
-                        ndarray[int64_t] labels,
+def group_last_{{name}}({{c_type}}[:, :] out,
+                        int64_t[:] counts,
+                        {{c_type}}[:, :] values,
```

Comment:
Can values be const?

Reply:
I wasn't able to compile that with cython.

I suspect that cython does not support const memory views with type object and object is one of the types used for {{name}}.